### DISCUSSION TITLE
Suggestion for CAM-Nor meeting Friday 28th May 2021

### DISCUSSION THREAD

**Discussion by oyvindseland** [NorESMhub/CAM#68] [Team Posts]
The original plan was to have a meeting 27th May, but I know that some of you have a KeyClim WP leader meeting at the same time. My suggestion is then 28th May 10-11.30 am. 

Suggested agenda, please add more points as needed
-Policy for branching of CAM-dev for development in projects, e.g. KeyClim, Forces and acceptance of pull requests in the branches
-Pull requests
-Issues

**Comment by oyvindseland** [comment]
I have received a suggestion that we move the meeting to 1 pm the same day

**Comment by DirkOlivie** [comment]
Both Thu 27 13:00 and Fri 28 10:00 would be fine for me.

**Comment by oyvindseland** [comment]
is Thursday afternoon also fine for the rest of you?

**Comment by annlew** [comment]
Hi,


Thursday 27/5 after 14.00 or Friday 28/5 after 13.00 would work for me.


Cheers,

Anna

________________________________

**Comment by oyvindseland** [comment]
Outdated discussion
